<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumperEditorWidget</class>
 <widget class="QWidget" name="JumperEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_jumperInfo">
         <property name="horizontalSpacing">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Imię</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Nazwisko</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Kod kraju</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Siła wybicia</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Technika wybicia</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Technika lotu</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Styl lotu</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Styl lądowania</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Forma</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_surname">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_country">
           <item>
            <widget class="QLineEdit" name="lineEdit_countryCode">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_country">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_countryFlag">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>IMG</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_takeoffPower">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_takeoffTechnique">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_flightTechnique">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_landingStyle">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_form">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_flightStyle">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 4px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
           </property>
           <item>
            <property name="text">
             <string>Styl V (0)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nowoczesne V (1)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Szerokie V (2)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Styl H (3)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_betweenJumperInfoAndCharacteristics">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>184</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_characteristicsEditor">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
